exercises/diamond/Example.cs

```diff
@@ -1,29 +1,32 @@
 ﻿using System;
 using System.Linq;
 
-public static class Diamond
+namespace Diamond
```

Comment:
Could you remove the namespace? We don't use namespaces in any of the exercises to make things as simple and clean as possible.